instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/datasource/JdbcTelemetryBuilder.java

```diff
@@ -42,12 +43,23 @@ public JdbcTelemetryBuilder setStatementSanitizationEnabled(boolean enabled) {
     return this;
   }
 
+  /**
+   * Sets whether to augment sql query with comment containing the tracing information. See <a
+   * href="https://google.github.io/sqlcommenter/">sqlcommenter</a> for more info.
+   */
+  @CanIgnoreReturnValue
+  public JdbcTelemetryBuilder setEnableSqlCommenter(boolean sqlCommenterEnabled) {
```

Comment:
@trask WDYT is adding it here fine or should I move this into `Experimental` class?